[2022-02-15 03:15:26,071] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.test.my_new_project.fct_orders manual__2022-02-15T03:14:06.780838+00:00 [queued]>
[2022-02-15 03:15:26,088] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.test.my_new_project.fct_orders manual__2022-02-15T03:14:06.780838+00:00 [queued]>
[2022-02-15 03:15:26,089] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:15:26,090] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 03:15:26,090] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:15:26,107] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): test.my_new_project.fct_orders> on 2022-02-15 03:14:06.780838+00:00
[2022-02-15 03:15:26,113] {standard_task_runner.py:52} INFO - Started process 1409 to run task
[2022-02-15 03:15:26,118] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'test.my_new_project.fct_orders', 'manual__2022-02-15T03:14:06.780838+00:00', '--job-id', '116', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmp23277w9z', '--error-file', '/tmp/tmp4zqn275o']
[2022-02-15 03:15:26,120] {standard_task_runner.py:77} INFO - Job 116: Subtask test.my_new_project.fct_orders
[2022-02-15 03:15:26,271] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.test.my_new_project.fct_orders manual__2022-02-15T03:14:06.780838+00:00 [running]> on host 459d09dd7c86
[2022-02-15 03:15:26,386] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=test.my_new_project.fct_orders
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T03:14:06.780838+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T03:14:06.780838+00:00
[2022-02-15 03:15:26,392] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 03:15:26,394] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json test --target dev --models fct_orders             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 03:15:26,410] {subprocess.py:85} INFO - Output:
[2022-02-15 03:15:35,701] {subprocess.py:89} INFO - 03:15:35  Running with dbt=1.0.1
[2022-02-15 03:15:36,216] {subprocess.py:89} INFO - 03:15:36  Found 8 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 03:15:36,226] {subprocess.py:89} INFO - 03:15:36
[2022-02-15 03:15:36,230] {subprocess.py:89} INFO - 03:15:36  [WARNING]: Nothing to do. Try checking your model configs and model specification args
[2022-02-15 03:15:36,889] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 03:15:36,938] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=test.my_new_project.fct_orders, execution_date=20220215T031406, start_date=20220215T031526, end_date=20220215T031536
[2022-02-15 03:15:37,016] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 03:15:37,056] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
